env/packages.yml

```diff
@@ -71,6 +71,9 @@
 
     - name: Compile gRPC and its dependencies
       shell: make -j{{ ansible_processor_vcpus }} chdir=/tmp/grpc
+      environment:
+        - CC: gcc-6
+        - CXX: g++-6
```

Comment:
Does it link correctly to BESS, which is built with a compiler other than g++-6? I vaguely remember g++ 4.8 has a different ABI, causing link issues to objects with newer ABIs.

Maybe it's okay for libraries to be built with newer ABIs than the program. Maybe the other way around...